src/js/menu/menu-button.js

```diff
@@ -113,11 +130,34 @@ class MenuButton extends ClickableComponent {
    */
   createEl() {
     return super.createEl('div', {
-      className: this.buildCSSClass()
+      className: this.buildWrapperCSSClass()
+    }, {
     });
   }
 
   /**
+   * Allow sub components to stack CSS class names for the wrapper element
+   *
+   * @return {string}
+   *         The constructed wrapper DOM `className`
+   */
+  buildWrapperCSSClass() {
```

Comment:
I just realized that the change to this `buildWrapperCSSClass` means that the elements lose out on having something like `vjs-captions-button` applied to the main wrapper element.

I think we want to return something like 
```js
return `${buttonClass} ${super.buildCSSClass()} ${this.buildCSSClass()}`;
```
instead, since the components are generally already set up with `buildCSSClass` to return the correct thing.

I'll open an issue for this.